.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+- Files changed
+- Dependencies if any (on other issues/PRs)
+- Basic tests done to validate
+- Conflicts if any (describe the reason for conflict)
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe):
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md) guidelines
+- [ ] Lint and unit tests pass locally with my changes
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] My changes generate no new warnings
```

Comment:
This is a little ambiguous -- is this browser console warnings, build warnings, something else?

Reply:
It is for if there is some warning while building.

Reply:
I think generally, this would be covered with passing tests, except for when there's a new console warning in the browser. I agree it could be a little more specific, but I also don't want a contributor to be blocked on this point if they're not sure what this means or that their PR would be rejected if they don't check this box. Kind of indifferent here - I'm OK with "If applicable, my changes generate no new console warnings," or just removing this altogether. I don't typically run a build on the project before submitting a PR and wouldn't expect a contributor to run a build before their changes and compare warnings with after their changes.

Reply:
Noted. I will remove it.